packages/components/psammead-caption/src/index.jsx

```diff
@@ -36,6 +36,9 @@ const Caption = styled.figcaption`
   @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
     padding: ${GEL_SPACING} 0;
   }
+  & > p:last-child {
+    padding-bottom: 0;
+  }
   & i {
```

Comment:
Could you please also update the Storybook story for `Caption` to have an example with multiple paragraphs? https://github.com/bbc/psammead/blob/47d91630e688bc6d854d0991ea74fa6a7c2ba523/packages/components/psammead-caption/src/index.stories.jsx#L43-L51